src/diffusers/loaders/lora.py

```diff
@@ -927,6 +929,23 @@ def fuse_lora(
                 Controls how much to influence the outputs with the LoRA parameters.
             safe_fusing (`bool`, defaults to `False`):
                 Whether to check fused weights for NaN values before fusing and if values are NaN not fusing them.
+            adapter_names (`List[str]`, *optional*):
+                Adapter names to be used for fusing. If nothing is passed, all active adapters will be fused.
+                Whether to check fused weights for `NaN` values before fusing and if values are `NaN`, then don't fuse
+                them.
```

Comment:
Seems to be a residue for `safe_fusing`?

Reply:
nice catch!